curriculum/challenges/english/25-front-end-development/lecture-working-with-regular-expressions/6733c5f20cc9584cada942a4.md

```diff
@@ -53,13 +65,16 @@ console.log("freecoooooooodecamp".replace(regex, "paidcodeworld"));
 
 We're getting `paidcodeworld` as our result. We want to preserve the number of `o`'s, so we need to reuse what was captured by the regular expression.
 
-This is where a backreference comes in. Instead of hardcoding the "`code`" portion of our replacement string, we can reference the captured group directly.
+This is where a backreference comes in. Instead of hardcoding the `code` portion of our replacement string, we can reference the captured group directly.
 
 In a replace call, you achieve a backreference by using a dollar sign (`$`) followed by the number of the capture group to use.
 
 In our case, that would be `$1`, since code is captured in the first capture group.
```

Comment:
```suggestion
In a `replace` call, you achieve a backreference by using a dollar sign (`$`) followed by the number of the capture group to use. In our case, that would be `$1`, since `code` is captured in the first capture group:
```